### PR TITLE
feat: add party migration

### PR DESCRIPTION
**Pull request by blushi**
## Description

Needed for regen-network/regen-registry#1423

As discussed, we'll be disabling row level security on `party` table manually on staging cc/ @wgwz 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted `dev` branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: sql/V2_16__party_grant.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+GRANT UPDATE (wallet_id) ON party TO app_user;
+ALTER TABLE party DROP CONSTRAINT check_image;
```
Comment (blushi) [inline sql/V2_16__party_grant.sql:2]: `check_image` was checking that if the party is an organization then `image` is not null, which is not the case anymore per latest design, image always being optional

**Comment by wgwz** [comment]
I suppose that since we only want this in staging and it's just temporary, we should just deploy this branch to staging and not merge to dev. Since dev eventually ends up merged into production. And then we just need a todo, to revert the effect of this migration. WDYT? @blushi

**Comment by blushi** [comment]
@wgwz we do want those changes on prod too, the only thing on prod we don't want is disabling the row level security policy.

**Comment by wgwz** [comment]
@blushi gotcha, makes sense. thanks for clarifying that.

**Pull request merged**